Bottom: 35d57b8123a5e38613688dda6e9fe4d28cb11133
Top:    89a781b5712cca5d0c199ce0a647a8f910fdfde3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 17:52:15 +0100

Refresh of move-arc-from-included-angle

---

diff --git a/src/misc.c b/src/misc.c
index ec37c3f..10796fc 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -330,7 +330,6 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
                                          Coord *cx, Coord *cy, Coord *radius,
                                          Angle *start_angle, Angle *delta_angle)
 {
-  Cardinal next_n;
   Coord hx, hy;
   double p_to_h_dist;
   double c_to_h_dist;
@@ -377,7 +376,7 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
                     (p1->Y - *cy) * (p1->Y - *cy));
 
   if (start_angle != NULL)
-    *start_angle = TO_DEGREES (atan2 ((p1->Y - *cy), -(p1->X - *cx)));
+    *start_angle = atan2 ((p1->Y - *cy), -(p1->X - *cx)) / M180;
 
   if (delta_angle != NULL)
     *delta_angle = included_angle;
